java/google/registry/flows/custom/DomainCreateFlowCustomLogic.java

```diff
@@ -92,6 +95,17 @@ public BeforeResponseReturnData beforeResponse(BeforeResponseParameters paramete
      */
     public abstract int years();
 
+    /**
+     * The ID of the validated signed mark.
+     *
+     * <p>If a signed mark was not supplied this value will be null</p>
+     */
+    @Nullable
+    public abstract String signedMarkId();
+
+    /** The time at which this flow was executed */
+    public abstract DateTime transactionTime();
```

Comment:
It's not strictly necessary to pass along transactionTime because Domain Create is a transactional flow, so you can always access the transaction time using ofy().getTransactionTime().